File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.java
import java.util.Arrays;

class Solution {
    public int solution(int[] arr1, int[] arr2) {
        if (arr1.length > arr2.length){
            return 1;
        } else if (arr1.length < arr2.length) {
            return -1;
        } else {
            return Integer.compare(Arrays.stream(arr1).sum(), Arrays.stream(arr2).sum());
        }
    }
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.kt
class Solution {
    fun solution(arr1: IntArray, arr2: IntArray): Int {
        return when {
            arr1.size < arr2.size -> -1
            arr1.size > arr2.size -> 1
            else -> arr1.sum().compareTo(arr2.sum())
        }
    }
}